### ISSUE TITLE
Consider type.today's Cyrillic review

### ISSUE THREAD

**Issue by thlinard** [stipub/stixfonts#220]
https://type.today/en/journal/transitional#StixTwoText

**Comment by davidmjones** [comment]
Thank you for this!  I'll forward it to STI Pub for their consideration.

@tiroj Aren't the math fonts supposed to be on Google as well?  I could have sworn I saw them there at one point.

**Comment by pkra** [comment]
> Aren't the math fonts supposed to be on Google as well? I could have sworn I saw them there at one point.

My browser history has https://fonts.google.com/specimen/STIX+Two+Math?query=STIX#glyphs which now gives a 404 (and nothing to be found when searching google fonts). But I'm quite certain this was their location.

**Comment by thlinard** [comment]
Due to https://github.com/google/fonts/issues/3773

**Comment by tiroj** [comment]
Re. the Cyrillic. During development of the STIX Two Cyrillic design, it was iteratively reviewed by Maxim Zhukov, former typographic coordinator at the United Nations and acknowledged expert on Cyrillic type design and typography. We’ve worked closely with Maxim on many Cyrillic projects over more than twenty years, and I am confident in his judgements and in the knowledge we’ve derived from that experience. The approach that Maxim favours differs from that promoted by Ilya Ruderman and his colleagues at CSTM, which tends to derive more directly from Latin with different proportional results.
_____

@davidmjones The Math font was on Google Fonts—actually, before the Text fonts—but then was removed pending proper support for math fonts in the GF API.

**Comment by thlinard** [comment]
There is one comment that comes up often in their reviews and example screenshots: "The curvature above descenders in Цц Щщ Дд was really unnecessary."

As far as I know, this is indeed in keeping with the established tradition of the Civil Type. But if Maxim Zhukov didn't see a problem, if he found this deviation from tradition acceptable, I would like to know where his tolerance comes from, if it's documented anywhere.

**Comment by thlinard** [comment]
They elaborated on their position [on Twitter](https://twitter.com/TypeTodayNews/status/1455478274080002049):

> We don't say it's always a mistake, sometimes it's right to keep the curvature, but only when all the stroke intersections have it. The descender is actually a part of the horizontal stroke, so there is no serif needed at the end of the vertical stroke (Like in Literata typeface)

![FDP8SASWUAMyYy8](https://user-images.githubusercontent.com/13808824/140503512-fbbabab2-42da-447b-b9af-bf2b128a7c30.jpg)

**Comment by tiroj** [comment]
Yes, bracketing of the transition of the second vertical to the spike is unnecessary. Sometimes, it assists the ‘colour’ though, especially if serif bracketing is pronounced elsewhere; this is also why the top of the spike sometimes angles down, rather than being flat as in those examples.
